tests/unit/test_validation.py

```diff
@@ -63,3 +67,41 @@ def test_validate_columns_and_metrics(testcase):
     else:
         with pytest.raises(ValueError):
             validate_evaluation_modes(test_dataset, testcase.metrics)
+
+
+column_maps = [
+    {
+        "question": "query",
+        "answer": "rag_answer",
+        "contexts": "rag_contexts",
+        "ground_truths": "original_answer",
+    },  # all columns present
+    {
+        "question": "query",
+        "answer": "rag_answer",
+    },  # subset of columns present
+]
```

Comment:
is there any more testcases that are missing?

Reply:
This is fine.